x-pack/plugins/beats_management/common/config_schemas_translations_map.ts

```diff
@@ -114,19 +114,19 @@ export const supportedConfigLabelsMap = new Map<string, string>([
   [
     'metricbeatModuleConfig.period.error',
     i18n.translate('xpack.beatsManagement.metricbeatModuleConfig.periodErrorMessage', {
-      defaultMessage: 'Invalid Period, must be formatted as `10s` for 10 seconds',
+      defaultMessage: 'Invalid format. Use `10s` for 10 seconds, with no ending period.',
```

Comment:
@gchaps In this case, I think it's better to say "invalid period" since the code tests the actual period specified, not just the format. Mind if I revert to the original?